ext/ftp/config.m4

```diff
@@ -3,21 +3,14 @@ PHP_ARG_ENABLE([ftp],
   [AS_HELP_STRING([--enable-ftp],
     [Enable FTP support])])
 
-PHP_ARG_WITH([openssl-dir],
-  [OpenSSL dir for FTP],
-  [AS_HELP_STRING([[--with-openssl-dir[=DIR]]],
-    [FTP: openssl install prefix])],
-  [no],
-  [no])
```

Comment:
After thinking about this a bit more, I don't think we can drop these, because they're the only way to force enable SSL support if you only want to build the ftp ext (without the openssl ext). This should probably be something like `--with-ftp-ssl` similar to the imap option.

But to avoid BC break for 7.4, we should probably just keep this as-is and just change the message to not talk about prefixes.